### PR TITLE
Issue 59 two card slider

### LINKED ISSUE #59: Slider: Two card variation
As we have removed the block config from sliders, we need to add a class to handle two cards in sliders.

https://www.caesars.com/careers/apply/day-of-hiring
Screen Shot 2023-05-10 at 8 47 47 AM

### PR DESCRIPTION
**Pull request by swetabar**
Fix #59 

## 🔗 Test URLs:

- Before: https://main--caesars-careers--hlxsites.hlx.page/drafts/swbarman/slider-two-card
- After: https://issue-59-two-card-slider--caesars-careers--hlxsites.hlx.page/drafts/swbarman/slider-two-card

## 📝 Description:

Screen Shot 2023-05-10 at 3 48 36 PM

### CODE REVIEW

**Comment by tmathern** [comment]
The sliding doesn't work for me on 2 cards, it keeps bouncing back to the first one and won't display the second one.

**Review by tmathern — CHANGES_REQUESTED** [review]
Sliding issue on mobile

File: careers/blocks/slider/slider.css [hunk 1]
```diff
@@ -278,6 +282,11 @@
   .slider.spacious .card-wrapper {
     gap: 50px;
   }
+
+  .slider.max-visible-2.block .card-wrapper {
+    width: calc(100%/2 - 4px);
```
Comment (tmathern) [inline careers/blocks/slider/slider.css:287]: Isn't 100%/2 the same as 50%?

Comment (swetabar) [reply careers/blocks/slider/slider.css:287]: Yes. Keeping that calculation for clarity. For 3 cards, it's 100%/3.

**Pull request merged**